Makefile

```diff
@@ -9,6 +9,8 @@ type:
 .PHONY: multi
 multi:
 	-python -m pyright src/gretel_trainer/relational tests/relational/
+	# vvv Do we need this...?
+	# pip install -e .
```

Comment:
I ran this manually a few time to get this make step to work, does it need to be enshrined forever or will the automated build be okay without it?

Reply:
I don't think it should be part of these make targets, but at some point we should consider some automation around the "welcome to Trainer" experience for devs/contributors.

The [CI build action](https://github.com/gretelai/trainer/blob/main/.github/workflows/build.yml#L17-L26) provides a good example of what needs to happen. (Maybe devs would want some virtualenv step.)

(Also just FYI as you can see in the build action linked above, in CI we run `make type` and `make test`, which cover the entire repo; these `multi[lint]` targets are just to speed up the local dev feedback loop while working exclusively on relational stuff.)

Reply:
Removed via Mike's branch.